lib/neo4j/rake_tasks/server_manager.rb

```diff
@@ -116,10 +116,16 @@ def config_auth_enabeled!(enabled)
       def config_port!(port)
         puts "Config ports #{port} (HTTP) / #{port - 1} (HTTPS) / #{port - 2} (Bolt)"
 
-        if server_version_greater_than?('3.0.0')
+        if server_version_greater_than?('3.0.9')
           # These are not ideal, perhaps...
           modify_config_file('dbms.connector.https.enabled' => false,
                              'dbms.connector.http.enabled' => true,
+                             'dbms.connector.http.listen_address' => "localhost:#{port}",
+                             'dbms.connector.https.listen_address' => "localhost:#{port - 1}",
+                             'dbms.connector.bolt.listen_address' => "localhost:#{port - 2}")
+        elsif server_version_greater_than?('3.0.0')
```

Comment:
If occurs to me that this won't work correctly for Neo4j 3.0.0.  Perhaps we should change the method to be `server_version_greater_than_or_equal_to?` (or `server_version_gte?`)?  That should make the code cleaner too